x-pack/plugins/canvas/public/expression_types/arg_types/series_style/simple_template.js

```diff
@@ -33,18 +34,34 @@ export const SimpleTemplate = props => {
       {!color || color.length === 0 ? (
         <Fragment>
           <EuiFlexItem grow={false}>
-            <label>Color&nbsp;</label>
+            <label>
+              <FormattedMessage
+                id="xpack.canvas.expressionTypes.seriesStyle.colorIsEmptyLabel"
+                defaultMessage="Color"
+              />
+              &nbsp;
+            </label>
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiLink onClick={() => handlePlain('color', '#000000')}>
-              Auto <EuiIcon type="bolt" />
+              <FormattedMessage
+                id="xpack.canvas.expressionTypes.seriesStyle.autoLinkLabel"
```

Comment:
xpack.canvas.expressionTypes.seriesStyle => xpack.canvas.seriesStyle for all ids in this file